templates/js/sso-tab-with-obo-flow/package.json.tpl

```diff
@@ -13,25 +14,28 @@
     "axios": "^0.21.1",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
-    "react-router-dom": "^6.8.0",
-    "react-scripts": "^5.0.1"
+    "react-router-dom": "^6.8.0"
   },
   "devDependencies": {
     "@types/node": "^18.0.0",
     "@types/react": "^18.0.0",
     "@types/react-dom": "^18.0.0",
     "@types/react-router-dom": "^5.3.3",
     "concurrently": "^8.2.2",
-    "env-cmd": "^10.1.0"
+    "env-cmd": "^10.1.0",
+    "typescript": "^4.1.2",
```

Comment:
We don't need the typescript dependency in a JS project.

Reply:
Right, I'll remove it.